.github/workflows/swift-toolchain.yml

```diff
@@ -2779,6 +2775,59 @@ jobs:
+          Move-Item ${env:SDKROOT}/usr/lib/swift/windows/FoundationNetworking.lib ${env:SDKROOT}/usr/lib/swift/windows/x86_64/
+          Move-Item ${env:SDKROOT}/usr/lib/swift/windows/_FoundationICU.lib ${env:SDKROOT}/usr/lib/swift/windows/x86_64/
+          Move-Item ${env:SDKROOT}/usr/lib/swift/windows/FoundationEssentials.lib ${env:SDKROOT}/usr/lib/swift/windows/x86_64/
+          Move-Item ${env:SDKROOT}/usr/lib/swift/windows/FoundationInternationalization.lib ${env:SDKROOT}/usr/lib/swift/windows/x86_64/
+
+      - name: Move host runtime DLLs to the runtime binary directory.
+        run: |
+          Remove-Item -Path ${env:SDKROOT}/usr/lib/swift/dispatch -Recurse -Force
+          Remove-Item -Path ${env:SDKROOT}/usr/lib/swift/os -Recurse -Force
+          Remove-Item -Path ${env:SDKROOT}/usr/lib/swift/Block -Recurse -Force
+          Remove-Item -Path ${env:SDKROOT}/usr/lib/swift/_foundation_unicode -Recurse -Force
+          Remove-Item -Path ${env:SDKROOT}/usr/lib/swift/_FoundationCShims -Recurse -Force
+
+          $RTLPath = cygpath -w "${{ github.workspace }}/BinaryCache/Developer/SDKs/Windows.sdk/usr/bin"
+          mkdir $RTLPath
+          Move-Item ${env:SDKROOT}/usr/bin/BlocksRuntime.dll  $RTLPath
+          Move-Item ${env:SDKROOT}/usr/bin/Foundation.dll  $RTLPath
+          Move-Item ${env:SDKROOT}/usr/bin/FoundationEssentials.dll  $RTLPath
+          Move-Item ${env:SDKROOT}/usr/bin/FoundationInternationalization.dll  $RTLPath
+          Move-Item ${env:SDKROOT}/usr/bin/FoundationNetworking.dll  $RTLPath
+          Move-Item ${env:SDKROOT}/usr/bin/FoundationXML.dll  $RTLPath
+          Move-Item ${env:SDKROOT}/usr/bin/_FoundationICU.dll  $RTLPath
+          Move-Item ${env:SDKROOT}/usr/bin/dispatch.dll  $RTLPath
+          Move-Item ${env:SDKROOT}/usr/bin/swiftCRT.dll  $RTLPath
+          Move-Item ${env:SDKROOT}/usr/bin/swiftCore.dll  $RTLPath
+          Move-Item ${env:SDKROOT}/usr/bin/swiftDispatch.dll  $RTLPath
+          Move-Item ${env:SDKROOT}/usr/bin/swiftDistributed.dll  $RTLPath
+          Move-Item ${env:SDKROOT}/usr/bin/swiftObservation.dll  $RTLPath
+          Move-Item ${env:SDKROOT}/usr/bin/swiftRegexBuilder.dll  $RTLPath
+          Move-Item ${env:SDKROOT}/usr/bin/swiftRemoteMirror.dll  $RTLPath
+          Move-Item ${env:SDKROOT}/usr/bin/swiftSwiftOnoneSupport.dll  $RTLPath
+          Move-Item ${env:SDKROOT}/usr/bin/swiftSynchronization.dll  $RTLPath
+          Move-Item ${env:SDKROOT}/usr/bin/swiftWinSDK.dll  $RTLPath
+          Move-Item ${env:SDKROOT}/usr/bin/swift_Builtin_float.dll  $RTLPath
+          Move-Item ${env:SDKROOT}/usr/bin/swift_Concurrency.dll  $RTLPath
+          Move-Item ${env:SDKROOT}/usr/bin/swift_Differentiation.dll  $RTLPath
+          Move-Item ${env:SDKROOT}/usr/bin/swift_RegexParser.dll  $RTLPath
+          Move-Item ${env:SDKROOT}/usr/bin/swift_StringProcessing.dll  $RTLPath
+          Move-Item ${env:SDKROOT}/usr/bin/swift_Volatile.dll  $RTLPath
```

Comment:
Globbing `*.dll` seems better than listing them. New libraries are added periodically, so it becomes annoying to maintain.

Reply:
+1